downloads.html

```diff
@@ -32,10 +32,31 @@
         <div id="installer-container">
             <h2>Installers</h2>
             <p>For the normal minecraft launcher and MultiMC</p>
-
-            <!-- todo: dynamically determine the version either through parsing the maven or the meta -->
-            <a href="https://maven.legacyfabric.net/net/legacyfabric/fabric-installer/0.11.1/fabric-installer-0.11.1.exe"><button class="download-button">Download .exe <br> (for windows)</button></a>
-            <a href="https://maven.legacyfabric.net/net/legacyfabric/fabric-installer/0.11.1/fabric-installer-0.11.1.jar"><button class="download-button">Download .jar <br> (universal)</button></a>
+            <a href="https://maven.legacyfabric.net/net/legacyfabric/fabric-installer/0.11.1/fabric-installer-0.11.1.exe" id="installer-exe"><button class="download-button">Download .exe <br> (for windows)</button></a>
+            <a href="https://maven.legacyfabric.net/net/legacyfabric/fabric-installer/0.11.1/fabric-installer-0.11.1.jar" id="installer-jar"><button class="download-button">Download .jar <br> (universal)</button></a>
             <a href="https://github.com/Legacy-Fabric/multimc-instance-creator/releases"><button class="download-button">MultiMC <br> instances</button></a>
 </body>
+<script>
+    var fallback = "0.11.1"
+    function setLatestMeta() {
+        var latest
+        fetch("https://meta.legacyfabric.net/v2/versions/installer")
```

Comment:
this breaks internet explorer

Reply:
Amazing

Reply:
if a browser doesn't support the javascript, it's still hardcoded in anyways so it'll still work even if its not the latest version